app/threads/client/components/ThreadView.tsx

```diff
@@ -69,9 +69,9 @@ const ThreadView = forwardRef<Element, ThreadViewProps>(({
 				<VerticalBar.Header>
 					<VerticalBar.Icon name='thread' />
 					<VerticalBar.Text dangerouslySetInnerHTML={{ __html: title }} />
-					{hasExpand && <VerticalBar.Action aria-label={expandLabel} name={expandIcon} onClick={handleExpandActionClick} />}
+					{hasExpand && <VerticalBar.Action title={t('Expand')} aria-label={expandLabel} name={expandIcon} onClick={handleExpandActionClick} />}
 					<VerticalBar.Actions>
-						<VerticalBar.Action aria-label={followLabel} name={followIcon} onClick={handleFollowActionClick} />
+						<VerticalBar.Action title={t(following ? 'Unfollow_message' : 'Follow_message')} aria-label={followLabel} name={followIcon} onClick={handleFollowActionClick} />
```

Comment:
you can use the value applied on the aria-label, and since you are using title, no need to use aria please remove ;)

Reply:
@ggazzo Made the changes :)